tests/nightly/jenkins-nightly-run.sh

```diff
@@ -61,7 +62,11 @@ testcases=("${@:-${DEFAULT_TESTCASES[@]}}")
 # we will be running or similar mechanism.
 VCH_BUILD=${VCH_BUILD:-${DEFAULT_VCH_BUILD}}
 VCH_BRANCH=${VCH_BRANCH:-${DEFAULT_VCH_BRANCH}}
-input=$(gsutil ls -l gs://vic-engine-builds/${VCH_BRANCH}${VCH_BRANCH:+/}${VIC_BINARY_PREFIX}${VCH_BUILD} | grep -v TOTAL | sort -k2 -r | head -n1 | xargs | cut -d ' ' -f 3 | cut -d '/' -f 4)
+ARTIFACT_BUCKET=${ARTIFACT_BUCKET:-${DEFAULT_ARTIFACT_BUCKET}}
+input=$(gsutil ls -l gs://${ARTIFACT_BUCKET}/${VCH_BRANCH}${VCH_BRANCH:+/}${VIC_BINARY_PREFIX}${VCH_BUILD} | grep -v TOTAL | sort -k2 -r | head -n1 | xargs | cut -d ' ' -f 3 | cut -d '/' -f 4)
+constructed_url="https://storage.googleapis.com/${ARTIFACT_BUCKET}/${VCH_BRANCH}${VCH_BRANCH:+/}${input}"
+url="${ARTIFACT_URL:-${constructed_url}}"
```

Comment:
To be consistent with the pattern this is mimicking, can we use `ARTIFACT_URL="${ARTIFACT_URL:-${constructed_url}}"`?